src/controls/adaptiveCardHost/AdaptiveCardHost.tsx

```diff
@@ -0,0 +1,232 @@
+
+
+import { PartialTheme, Theme, ThemeProvider } from '@fluentui/react-theme-provider';
+import { Action, AdaptiveCard, CardElement, CardObjectRegistry, ExecuteAction, GlobalRegistry, HostConfig, OpenUrlAction, SerializationContext, SubmitAction } from 'adaptivecards';
+import { Template } from 'adaptivecards-templating';
+import { CustomizerContext, ITheme } from 'office-ui-fabric-react';
+import * as React from 'react';
+import {
+    useCallback,
+    useEffect,
+    useRef
+} from 'react';
+import './AdaptiveCardHost.css';
+import { convertFromPartialThemeToTheme, createDarkTeamsHostConfig, createDefaultTeamsHostConfig, createHighContrastTeamsHostConfig, createSharePointHostConfig, initProcessMarkdown, setFluentUIThemeAsCSSVariables } from './AdaptiveCardHostHelpers';
+import { createDarkTeamsTheme, createDefaultTeamsTheme, createHighContrastTeamsTheme, getDefaultFluentUITheme, setFluentUIThemeAsHostCapability, useLocalFluentUI } from './fluentUI';
+import { IAdaptiveCardHostActionResult, IAdaptiveCardHostProps, AdaptiveCardHostThemeType } from './IAdaptiveCardHostProps';
+
+// Init Process Markdown
+initProcessMarkdown();
+
+export const AdaptiveCardHost = (props: IAdaptiveCardHostProps) => {
```

Comment:
@AJIXuMuK Is destructuring mandatory?
In general, I prefer to understand which variables come from the props and which do not, by deconstructing this advantage is lost. 
If it is mandatory to deconstruct it, I do it, otherwise I prefer to leave it like this.
What do I have to do? 🙂